CMakeLists.txt

```diff
@@ -153,7 +153,7 @@ SET(asm2wasm_SOURCES
 )
 ADD_EXECUTABLE(asm2wasm
                ${asm2wasm_SOURCES})
-TARGET_LINK_LIBRARIES(asm2wasm emscripten-optimizer ${all_passes} wasm asmjs support)
+TARGET_LINK_LIBRARIES(asm2wasm passes emscripten-optimizer wasm asmjs support)
```

Comment:
not needed. 

I was fiddling with the make files because I got some weird linking errors and forgot to back out this change. 

r